skglm/solvers/group_bcd_solver.py

```diff
@@ -95,6 +102,12 @@ def group_bcd_solver(X, y, datafit, penalty, w_init=None, Xw_init=None, p0=10,
             # inplace update of w and Xw
             _bcd_epoch(X, y, w, Xw, datafit, penalty, ws)
```

Comment:
This is harmless because of ws, but for clarity probably w[:n_features] should be passed